package.json

```diff
@@ -24,6 +24,7 @@
     "@emotion/styled": "^11.6.0",
     "firebase": "^9.5.0",
     "framer-motion": "^4.1.17",
+    "graphql": "^16.2.0",
```

Comment:
I think we don't need to add `graphql` as a dependency, we already have `@apollo/client` to consume any graphql apis.